test/unit/renderer/viz/expressions/buckets.test.js

```diff
@@ -65,6 +65,11 @@ describe('src/renderer/viz/expressions/buckets', () => {
 
                     expect(response).toEqual(expected);
                 });
+
+                it('should throw an error when not using an array for the breakpoint', () => {
```

Comment:
I think the util function `validateCompileTypeError` would be better, at least that will take care of calling `bindMetadata` for you

Reply:
👍 I'll have a look at it

Reply:
I applied your suggestions. Same results with less code, thx!